### ISSUE TITLE
Stunned shield

### ISSUE THREAD

**Issue by Madis0** [Minenash/Enhanced-Attack-Indicator#2]

**Comment by Minenash** [comment]
This was actually already incuded, kinda. While it did show, due to how I had the item cooldown done (which the shield uses for stun), if the player was holding any item in their main, then nothing showed up.

So what I changed was, instead of it seeing which hand has an item, and returning it's cooldown, it see's which item has a cooldown, and if either does, then it returns it. And the order of check has went from mainhand then offhand, to offhand to mainhand. The reason for the swap is to help prioritize shields, and if a player has a different item cooldown in it's offhand, it probably means theirs a weapon or block in their main hand, which don't have cooldowns anyways 

Basically went from:
***
Does mainhand have an item?
- Yes: return it's cooldown, if it doesn't have any, return 1 (which means doesn't show)
- No: Does offhand have an item?
   - Yes: return it's cooldown, if it doesn't have any, return 1 (which means doesn't show)
   - No: return 1
***
To:
***
Does offhand have an item with a cooldown?
- Yes: return the cooldown
- No: Does mainhave an item with a cooldown?
   - Yes: return cooldown
   - No: return 1
***

**Comment by Minenash** [comment]
[Fixed in this commit](https://github.com/Minenash/Enhanced-Attack-Indicator/commit/4a0fd18b4025776bdb5f9d39784a6a58757396d2)
